cards/git-realm/lib/git-settings.ts

```diff
@@ -0,0 +1,43 @@
+interface IndexerSettings {
+  repo: string;
+  basePath: string;
+  branchPrefix: string;
+  remote?: RemoteConfig;
+}
+import { RemoteConfig } from './git';
+
+import CardstackError from '@cardstack/core/error';
+import { AddressableCard } from '@cardstack/core/card';
+
+export function extractSettings(realmCard: AddressableCard): IndexerSettings {
+  let attributes = realmCard.attributes;
```

Comment:
@alexspeller AddressableCard.attributes is private. you'll need to use AddressableCard.value('repo'), etc to get values off the card.